build.gradle

```diff
@@ -24,54 +28,67 @@ repositories {
-    implementation group: 'com.google.code.findbugs', name: 'annotations', version: '3.+'
-    implementation group: 'com.google.apis', name: 'google-api-services-logging', version: 'v2+'
+
+
+    // Google dependencies - make sure we are using -jre and not android and use common bom
+    constraints {
+        implementation 'com.google.guava:guava:30.1.1-jre' // "-jre" for Java 8 or higher
+    }
+    implementation platform('com.google.cloud:libraries-bom:20.2.0')
+
+    implementation group: 'bio.terra', name: 'stairway', version: '0.0.51-SNAPSHOT'
+    implementation group: 'org.postgresql', name: 'postgresql', version: '42.2.18'
+
+    implementation group: 'ch.qos.logback.contrib', name: 'logback-json-classic', version: '0.1.5'
+    implementation group: 'ch.qos.logback.contrib', name: 'logback-jackson', version: '0.1.5'
+    implementation group: 'com.google.cloud', name: 'google-cloud-core'
+    implementation group: 'com.google.code.findbugs', name: 'annotations', version: '3.0.1'
+    implementation group: 'com.google.apis', name: 'google-api-services-logging', version: 'v2-rev20210507-1.31.0'
     implementation group: 'io.kubernetes', name: 'client-java', version: '10.0.0'
     implementation group: 'io.opencensus', name: 'opencensus-exporter-trace-stackdriver', version: opencensus
     // commons-dbcp2 and -lang3 versions are managed by the Spring Boot plugin
-    implementation group: 'org.apache.commons', name: 'commons-dbcp2'
-    implementation group: 'org.apache.commons', name: 'commons-lang3'
-    implementation group: 'org.apache.commons', name: 'commons-pool2'
-    implementation group: 'org.hashids', name: 'hashids', version: '1.+'
-    implementation group: 'org.liquibase' , name: 'liquibase-core', version: '3.10.0'
-    implementation group: 'org.slf4j', name: 'slf4j-api', version: '1.7.+'
-    implementation group: 'org.springframework' , name: 'spring-context'
-    implementation group: 'org.springframework' , name: 'spring-core'
-    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-data-jdbc'
-    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web'
+    implementation group: 'org.apache.commons', name: 'commons-dbcp2', version: '2.8.0'
+    implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.11'
+    implementation group: 'org.apache.commons', name: 'commons-pool2', version: '2.9.0'
+    implementation group: 'org.hashids', name: 'hashids', version: '1.0.3'
+    implementation group: 'org.liquibase' , name: 'liquibase-core', version: '4.2.1'
+    implementation group: 'org.slf4j', name: 'slf4j-api', version: '1.7.30'
+    implementation group: 'org.springframework' , name: 'spring-context', version: '5.3.7'
+    implementation group: 'org.springframework' , name: 'spring-core', version: '5.3.7'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-data-jdbc', version: '2.4.5'
```

Comment:
Where did you get all these version numbers? I don't know what's a real change and what isn't. I suppose `gradle dependencies` is the way to get them?